packages/legend-application/src/components/LambdaEditor.tsx

```diff
@@ -325,6 +325,9 @@ const LambdaEditorInline = observer(
         editorModel.updateOptions({ tabSize: TAB_SIZE });
         const error = parserError ?? compilationError;
         if (error?.sourceInformation) {
+          if (error.message === '') {
```

Comment:
Rethinking this I dont like us replacing the orginal value of the error. Could we may add a getter property to `EngineError` called nonEmptyMessage where we return either the orginal message if not empty and if empty `No error message provided by engine`.